cargo-concordium/src/main.rs

```diff
@@ -394,10 +512,19 @@ pub fn main() -> anyhow::Result<()> {
                         );
                     }
 
-                    eprintln!("   Writing schema to {}.", schema_out.display());
                     fs::write(schema_out, &module_schema_bytes)
                         .context("Could not write schema file.")?;
                 }
+                if let Some(schema_json_out) = schema_json_out {
+                    ensure!(
+                        schema_json_out.is_dir(),
+                        "The `--schema-expand-out` flag should point to an existing directory \
+                         (expected input `./my/path`)."
+                    );
+                    eprintln!("   Writing schema to {}.", schema_json_out.display());
```

Comment:
Not sure if we need it since we also print the below message in the `build.rs` file, but also don't mind if we print out too much here :)

    println!(
        "   Writing schema for {} to {}.",
        contract_name,
        out_path.display()
    );

Reply:
I agree, this should be removed. I removed it in the other place already, but forgot this.